File: examples/go/store.go
package main
import ("database/sql"; "os/exec")
const AWS_SECRET_ACCESS_KEY = "wJalr8UtnFEMI7K9MDENGbPxRfiCY2EX"
func find(db *sql.DB, name string) { db.Query("SELECT * FROM users WHERE n = '" + name + "'") }
func ping(h string) { exec.Command("sh", "-c", "ping "+h).Run() }

File: examples/cpp/store.cpp
const char* API_SECRET = "a7f3b9c2d4e6f8a1b3c5";
void find(const std::string& name) { query("SELECT * FROM t WHERE n = '" + name + "'"); }
void ping(const std::string& h) { system(("ping " + h).c_str()); }
